spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -139,6 +139,8 @@ def select_team
           puts "#{i + 1}) \"#{team['contentProvider']['name']}\" (#{team['contentProvider']['contentProviderId']})"
         end
 
+        raise "Running in non-interactive shell" if UI.interactive? == false
```

Comment:
I very much like the intent of this PR, however _spaceship_ doesn't have access to `FastlaneCore::UI`, as they're in separate subfolders, at least until now. Definitely something we could change, as we'd also need this for `credentials_manager`, at both points we don't fail the build if it's not interactive.